### DISCUSSION TITLE
Is react better than bootstrap for frontend design??

### DISCUSSION THREAD

**Discussion by Masterminddac** [twbs/bootstrap#38837] [General]
I keep seeing jobs that are looking for react designers 
I am very good with bootstrap

I want to know if I should switch and start using react or continue using bootstrap

**Comment by gerardnico** [comment]
React is a JavaScript library that models the dom while bootstrap is a ui/design library.

You can use React with Bootstrap.

Check the following library:

* https://react-bootstrap.netlify.app/

* https://reactstrap.github.io/?path=/story/home-installation--page
